src/components/NewMovie/NewMovie.tsx

```diff
@@ -48,6 +122,9 @@ export const NewMovie = () => {
             type="submit"
             data-cy="submit-button"
             className="button is-link"
+            disabled={checkRequired()
+              || (!validationUrl(movie.imdbUrl)
+              || !validationUrl(movie.imgUrl))}
```

Comment:
create a separate variable for it
```suggestion
            disabled={isDisable}
```